src/parser/mage/fire/modules/features/HotStreakWastedCrits.tsx

```diff
@@ -39,18 +44,21 @@ class HotStreakWastedCrits extends Analyzer {
 
   //When a spell that contributes towards Hot Streak is cast, get the event info to use for excluding the cleaves from Phoenix Flames on the damage event.
   //If a Hot Streak Contributor was cast then Pyromaniac didnt proc, so set it to false (Pyromaniac procs when Hot Streak is used, so if something was cast, then it didnt proc)
-  _onCast(event) {
+  _onCast(event: CastEvent) {
     this.lastCastEvent = event;
     this.pyromaniacProc = false;
```

Comment:
This is unused?

Reply:
it was just named wrong. should have been hasPyromaniacProc